src/Foundation/Site.php

```diff
@@ -25,7 +25,10 @@ public static function fromPaths(array $paths)
         date_default_timezone_set('UTC');
 
         if (! static::hasConfigFile($paths->base)) {
-            return new UninstalledSite($paths, $_SERVER['REQUEST_URI']);
+            return new UninstalledSite(
+                $paths,
+                Arr::get($_SERVER, 'REQUEST_URI')
```

Comment:
Is this change necessary for this PR?

Reply:
Without this change the CLI still fails, because the original error is that $_SERVER does not contain that key:

```
PHP Notice:  Undefined index: REQUEST_URI in /home/forge/nightly.flarum.site/nightly/vendor/flarum/core/src/Foundation/Site.php on line 28
```

This is **the** fix. But because in the code the URL cannot be NULL we needed deeper changes.

Reply:
Makes sense. Let's keep that then!